Makefile

```diff
@@ -56,42 +56,43 @@ api.env.test: guard-AUTH0_CLIENT_ID_TEST guard-AUTH0_CLIENT_SECRET_TEST api.clea
 	(cd api; sed -i 's|pipelinedashboard-projects|pipelinedashboard-projects-test|g' ./config.json)
 
 api.test:
-	(cd api; ./node_modules/.bin/cucumber-js ./tests/features --require ./tests --format ./node_modules/cucumber-pretty --tags ${TEST_TAGS})
+	(cd api; rm -fr coverage)
+	(cd api; npm test)
```

Comment:
should be `npm run test` this is why CI fails I think....

Reply:
I thought `test` was a special one? It works locally, but I can change it just in case. Thanks for the tip 🤓

Reply:
I did think that as I wrote, I am just used to using run for everything except start :)

Reply:
yeah probably better to have it in front of most things as its less likely to cause an issue in the future